### ISSUE TITLE
Media keys don't work anymore (OS X 10.11.6)

### ISSUE THREAD

**Issue by neoyoli** [JorritPosthuma/MixCloud-Desktop#15]
Until recently, as I recall it has been working.

**Comment by axsauze** [comment]
I can report that this doesn't work in 10.11.6 (El Capitan) either - I've added permissions in Privacy -> Accessibility, but it still doesn't work. Is there any further configuration required for this to work?

**Comment by TheSpyder** [comment]
It's possible Apple changed something. Now that Safari supports media keys directly, there is nobody left who is interested in maintaining this app. Contributions are welcome, of course.

**Comment by TheSpyder** [comment]
I'm actually removing the manual media key handling, I had it all updated to conform to the accessibility standards and then I realised it doesn't actually do anything with the modern Safari webview.

By removing the need for accessibility access perhaps it will start working for you? Build should be up soon.

**Comment by Impakt** [comment]
I'm on 10.12.6 and the media keys don't work. They don't work on the previous release either. I also tried adding the app in accessbility preferences

**Comment by uffou** [comment]
Made an electron MixClould app - support for tracks in the menubar, media controls and notifications https://github.com/uffou/MixCloud-Play
